File: Programmers/코딩테스트_입문/GangHyeok/Day03/최빈값 구하기.java
class Solution {
    public int solution(int[] array) {
        int answer = 0;
        int []arr = new int[1000];
        int max = 0;
        
        for(int i=0;i<array.length;i++){
            arr[array[i]]++;
        }
        
        for(int i=0;i<1000;i++){
            if(arr[i]>max){
                answer = i;
                max = arr[i];
            
            }
            else if(arr[i] == max){
                answer = -1;
            }
        }
        
        return answer;
    }
}

File: Programmers/코딩테스트_입문/GangHyeok/Day03/최빈값 구하기.js
function solution(array) {
  var answer = 0;
  
  let arr = [];
  let max = 0;
  

  
  for(let i of array){
      if(arr[i] == null){
          arr[i] = 1;
      }
      else{
          arr[i]++;    
      }
      
  }
  
  for(let i in arr){
      if(arr[i]){
          if(arr[i] > max){
          answer = i;
          max = arr[i]
      }
      else if(arr[i] == max){
          answer = -1;
      }    
      }
      
  }
  
  return parseInt(answer);
}